en/ui-system/components/engine/ui-material.md

```diff
@@ -49,13 +51,13 @@ Using a UI built-in material works the same as custom materials. However, there
 
     ![dissolve](ui-material/dissolve.png)
 
-5. To perform uniform assignment operations to custom materials, they can operate by obtaining the material on the Sprite. We provide different interfaces to deal with different operating conditions, as shown in the following code: **(Please pay attention to the difference Notes on the interface!)**
+8. If the user wants to assign a `uniform` to a custom material, he can do so by getting the customMaterial on the 2D rendered component. Different interfaces are provided to handle different cases of operations, as shown in the following code: (using Sprite as an example) **(please be sure to read the annotations of the different interfaces!)**
```

Comment:
```suggestion
8. If the user wants to assign a `uniform` to a custom material, it can be done by getting the customMaterial on the 2D renderer component. Creator provides different interfaces for different cases, see the following example code for Sprite:
```